test/test_bar.py

```diff
@@ -95,3 +95,25 @@ def test_bar_datazoom_inside():
     assert ': "inside"' in bar._repr_html_()
     assert ': "slider"' not in bar._repr_html_()
     bar.render()
+
+
+def test_bar_datazoom_both():
+    # bar dataZoom both-type
+    attr = ["{}月".format(i) for i in range(1, 13)]
+    v1 = [2.0, 4.9, 7.0, 23.2, 25.6, 76.7, 135.6, 162.2, 32.6, 20.0, 6.4, 3.3]
+    v2 = [2.6, 5.9, 9.0, 26.4, 28.7, 70.7, 175.6, 182.2, 48.7, 18.8, 6.0, 2.3]
+    bar = Bar("柱状图示例")
+    bar.add("蒸发量", attr, v1, mark_line=["average"], mark_point=["max", "min"])
+    bar.add("降水量", attr, v2, mark_line=["average"], mark_point=["max", "min"],
+            is_datazoom_show=True, datazoom_range=[20, 60], datazoom_type='both')
+    assert "dataZoom" in bar._repr_html_()
+    bar.render()
+
+    attr = ["{}天".format(i) for i in range(30)]
+    v1 = [random.randint(1, 30) for _ in range(30)]
+    bar = Bar("Bar - datazoom - both 示例")
+    bar.add("", attr, v1, is_datazoom_show=True, datazoom_type='both',
+            datazoom_range=[10, 25])
+    assert ': "inside"' in bar._repr_html_()
```

Comment:
html_content = bar._repr_html()
assert '..' in html_content
assert '..' in ....

it would use less time during test